libmbp/devices/asus.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2015  Andrew Gunnerson <andrewgunnerson@gmail.com>
+ * Copyright (C) 2016  Andrew Gunnerson <andrewgunnerson@gmail.com>
```

Comment:
These should all be `2015-2016`. Sorry to nitpick, but I can get in trouble (with my employer) if this is not done properly :)